src/services/GitServices/GitCommitGenService.ts

```diff
@@ -1,8 +1,8 @@
 import { spawnProcess } from "../OtherServices/SpawnService";
 import { logInfo, logError, logSuccess } from "../OtherServices/LogService";
-import { CheckIfArgIncludes } from "../OtherServices/CommandService";
 import { writeFileSync, existsSync } from "fs";
 import { LoadService } from "../OtherServices/LoadService";
+import { commitGen } from "../../config/gitlestial.config";
```

Comment:
my meaning is not importing the config file, but more like reading it from the directory on which user is running gitlestial command,

e.g,

test_app has gitlestial.config.js in the root directory.

so we can 
```
$ cd test_app
$ gitlestial commit-gen
```

it will trigger the same as 
```
$ gitlestial commit-gen --amount 100 --contributors test11@gmail.com,test111@gmail.com -am "just a test"
```
if the gitlestial.config.js has 
```
module.exports = {
  commitGen: {
    amount: 100,
    contributors: ["test11@gmail.com","test11@gmail.com"],
    message: "just a test",
  },
};
```

it's not a static import but a dynamic reading of a config file